week-01-reflections/lsilveirajunior-stat6863.md

```diff
@@ -2,10 +2,21 @@
 # Questions about Problems and Recipes
 
 
-
-[place your questions/answers here, and delete this line]
-
-
+[Course Structure Quiz, Problem 5]
+∗ Question (<lsilveirajunior-stat6863 >): Will the final exam have multiple choice questions similar to the exercises done during the 10-week program or new material will be added to it?
```

Comment:
This is not the correct Q/A format: The asterisk you're using does not render as a bullet point in Markdown, and there should not be angle brackets around your GitHub username